app/src/main/java/fr/free/nrw/commons/review/ReviewController.java

```diff
@@ -209,6 +215,16 @@ public void sendThanks(@NonNull Activity activity) {
                             .setProgress(0, 0, false)
                             .setOngoing(false)
                             .setPriority(NotificationCompat.PRIORITY_HIGH);
+                    /**
+                     * On tapping the notification it should take to visit the Thanks Log page
+                     * Thus adding a pending content to redirect to a url depending on the beta flavor*/
+                    String url;
+                    if (ConfigUtils.isBetaFlavour())
+                        url = BuildConfig.HOME_URL+"Special:Log/thanks";
+                    else url = THANKS_LOG;
+                    Intent browserIntent = new Intent(Intent.ACTION_VIEW , Uri.parse(url));
```

Comment:
We generally use If and elses with braces, please add them too:)

Reply:
Actually this if/else can be removed at all. You can just use BuildConfig.Home_URL variable. Is returns the correct URL according to flavour.

Reply:
@neslihanturan I have made the changes :+1: